### ISSUE TITLE
BTM: listtransactions returns an empty array

### ISSUE THREAD

**Issue by ghost** [jl777/SuperNET#108]
LISTTRANSACTIONS

curl --url "http://127.0.0.1:7778" --data "{\"coin\":\"BTM\",\"method\":\"listtransactions\",\"params\":[1, 9999999, [\"bMS5HSgJoJJemNGCEAVQerDFZMJuyw5d6p\"]]}"

RETURNS:
{"result":[],"tag":"14159682109545392018"}

All other calls work as stated in slack.

**Comment by jl777** [comment]
was your wallet unlocked?

**Comment by ghost** [comment]
Yes I unlocked it. If I attempt to run ./listtransactions without unlocking it, it would respond with:
{"error":"need to unlock wallet","tag":"17201538927344179581"}

I log in with my wallet which has my test transactions to that bM...6p address. 
Run the ./listtransactions above and it returns:
{"result":[],"tag":"14159682109545392018"}
